server/parser/parser.service.ts

```diff
@@ -4,37 +4,37 @@ import { ParagraphService } from "../paragraph/paragraph.service";
 import { SpeechService } from "../speech/speech.service";
 import { SpeechDocument } from "../speech/schemas/speech.schema";
 const md5 = require("md5");
-const nlp = require('compromise')
-nlp.extend(require('compromise-sentences'))
-nlp.extend(require('compromise-paragraphs'))
+const nlp = require("compromise");
+nlp.extend(require("compromise-sentences"));
+nlp.extend(require("compromise-paragraphs"));
 
-// TODO: regex for future rules
-// const alphabets = /([A-Za-z])/g;
-const prefixes = /(Mr|St|Mrs|Ms|Dr)[.]/g;
-const phdRegex = /Ph\.D\./g;
-// const suffixes = "(Inc|Ltd|Jr|Sr|Co)";
-// const starters = /(Mr|Mrs|Ms|Dr|He\s|She\s|It\s|They\s|Their\s|Our\s|We\s|But\s|However\s|That\s|This\s|Wherever)/g;
-// const acronyms = /([A-Z][.][A-Z][.](?:[A-Z][.])?)/g;
-// const websites = /[.](com|net|org|io|gov)/g;
+/* TODO: regex for future rules
```

Comment:
We can remove this, compromise already cover most of it.

Another solution for code quality is to add test cases for each of these class of problems.